buildchain/buildchain/iso.py

```diff
@@ -78,13 +78,44 @@ def task_populate_iso() -> types.TaskDict:
         'task_dep': [
             '_iso_mkdir_root',
             '_iso_render_bootstrap',
+            '_iso_add_node_manifest',
             '_iso_generate_product_txt',
             'images',
             'salt_tree',
             'packaging',
         ],
     }
 
+def task__iso_mkdir_samples() -> types.TaskDict:
+    """Create ISO_ROOT/samples"""
+    return helper.Mkdir(
+            directory=constants.ISO_ROOT/"samples", task_dep=['_iso_mkdir_root']
```

Comment:
minor/layout: in this file (see above) we usually indent with 4 space when continuing a line, so

```python
    return helper.Mkdir(
        directory=constants.ISO_ROOT/"samples", task_dep=['_iso_mkdir_root']
    ).task
```

would be more consistent